application/src/test/java/uk/gov/hmcts/net/ccd/definition/store/excel/SpreadSheetImportTest.java

```diff
@@ -212,6 +214,74 @@ public void userProfileIsNotStoredWhenImportFails() throws Exception {
             jdbcTemplate.queryForObject(GET_CASE_TYPES_COUNT_QUERY, Integer.class).intValue());
     }
 
+    /**
+     * API test for successful import of a valid Case Definition with Noc Config spreadsheet.
+     *
+     * @throws Exception On error running test
+     */
+    @Test
+    @Transactional
+    public void importValidDefinitionFileContainsNocConfig() throws Exception {
+
+        try (final InputStream inputStream =
+                 new ClassPathResource(EXCEL_FILE_NOC_CONFIG, getClass()).getInputStream()) {
+            MockMultipartFile file = new MockMultipartFile("file", inputStream);
```

Comment:
Two lines could be extracted to one nice method and used in all three tests.

Reply:
done